<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialDialogBase</class>
 <widget class="QWidget" name="MaterialDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Material</string>
  </property>
  <property name="sizeGripEnabled" stdset="0">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="HBoxLayout">
     <item>
      <spacer name="spacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>65</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="textLabel1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Store in C&amp;hapter</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>mCbChapter</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mCbChapter"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Material</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="mEditMaterial"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_2">
        <item>
         <spacer name="spacer3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>50</width>
            <height>16</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="textLabel2">
          <property name="text">
           <string>Pac&amp;kaged:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>mDiPerPack</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="mDiPerPack">
          <property name="maximum">
           <double>99999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
          <property name="precision" stdset="0">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mCbUnit"/>
        </item>
        <item>
         <widget class="QLabel" name="textLabel3">
          <property name="text">
           <string>per P&amp;ackage</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>mCbUnit</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Prices</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="spacer5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>240</width>
          <height>11</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="_4">
        <item row="1" column="1">
         <widget class="QLabel" name="textLabel4_2">
          <property name="text">
           <string>= Price of &amp;sale:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>mInSalePrice</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QDoubleSpinBox" name="mInSalePrice">
          <property name="maximum">
           <double>999999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
          <property name="maxValue" stdset="0">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QDoubleSpinBox" name="mInPurchasePrice">
          <property name="maximum">
           <double>999999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
          <property name="maxValue" stdset="0">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="_5">
          <item>
           <widget class="QLabel" name="textLabel1_2">
            <property name="text">
             <string>pl&amp;us</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>mInSaleAdd</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="mInSaleAdd">
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="maximum">
             <double>999.000000000000000</double>
            </property>
            <property name="label" stdset="0">
             <string/>
            </property>
            <property name="precision" stdset="0">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="textLabel4">
          <property name="text">
           <string>&amp;Purchase price:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>mInPurchasePrice</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>mEditMaterial</tabstop>
  <tabstop>mDiPerPack</tabstop>
  <tabstop>mCbUnit</tabstop>
  <tabstop>mInPurchasePrice</tabstop>
  <tabstop>mInSaleAdd</tabstop>
  <tabstop>mInSalePrice</tabstop>
  <tabstop>mCbChapter</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
